### DISCUSSION TITLE
How to search if I have a duplicate recipe

### DISCUSSION THREAD

**Discussion by pippo73** [TandoorRecipes/recipes#3368] [Ideas]
Hello, 
I would like to know if there is some way to find (and maybe delete) all the duplicate recipe I have in my DB.

Ciao to all

**Comment by vabene1111** [comment]
hi, there is currently no dedicated duplicate search.

**Reply by pippo73** [reply]
Is it possibile to rise a "request" in the todo list? ;-)

**Reply by vabene1111** [reply]
You can always write an issue for this, but this wont make implementing it any faster. 
Or do you mean you want to pay me to implement that?

**Reply by smilerz** [reply]
Are you looking to find recipes with the same name?  Or literally duplicate recipes, same name, same ingredients, etc?
